Bottom: ca9b255bce131fb11d7db36c3e128c58422a30a2
Top:    4d675a1c9f074e96202177d3bfa404a7ec27764e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 21:42:41 +0000

Refresh of draw-polygons-in-a-more-intell

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ecac2fc..b36b83f 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1330,7 +1330,7 @@ poly_callback (const BoxType * b, void *cl)
   PolygonType *polygon = (PolygonType *) b;
 
   set_layer_object_color (i->layer, (AnyObjectType *) polygon);
-  gui->graphics->draw_pcb_polygon (Output.fgGC, polygon, i->drawn_area);
+  hid_draw_pcb_polygon (Output.fgGC, polygon, i->drawn_area);
   return 1;
 }
 
@@ -1344,7 +1344,7 @@ poly_callback_no_clear (const BoxType * b, void *cl)
     return 0;
 
   set_layer_object_color (i->layer, (AnyObjectType *) polygon);
-  gui->graphics->draw_pcb_polygon (Output.fgGC, polygon, i->drawn_area);
+  hid_draw_pcb_polygon (Output.fgGC, polygon, i->drawn_area);
   return 1;
 }
